src/applications/vaos/components/review/Description.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { getTypeOfCare } from '../../utils/selectors';
+import { FLOW_TYPES, FACILITY_TYPES } from '../../utils/constants';
+import { lowerCase } from '../../utils/appointment';
+
+export default function Description({ data, flowType }) {
+  const typeOfCare = lowerCase(getTypeOfCare(data).name);
+  const description =
+    data.facilityType === FACILITY_TYPES.COMMUNITY_CARE
+      ? 'Community Care'
+      : typeOfCare;
+  const isDirectSchedule = flowType === FLOW_TYPES.DIRECT;
+
+  return (
+    <>
+      <h2 className="vads-u-margin-bottom--0 vads-u-margin-top--3 vads-u-font-size--h3">
+        You're {isDirectSchedule ? 'scheduling' : 'requesting'} a {description}{' '}
```

Comment:
Minor detail but I've been told that we should be using `’` instead of single quotes in copy.  BTW. I found the keyboard shortcut on Mac: `OPTION + SHIFT + ]`.